src/renderer/screens/exchange/Swap2/Form/FormSummary/SectionFees.js

```diff
@@ -17,14 +17,19 @@ import Text from "~/renderer/components/Text";
 const SectionFees = ({ swapTransaction }: { swapTransaction: SwapTransactionType }) => {
   const { t } = useTranslation();
   const { setDrawer } = React.useContext(context);
-  const { account, transaction } = swapTransaction;
+  const { transaction } = swapTransaction;
   const exchangeRate = useSelector(rateSelector);
-  const fromAccount = swapTransaction.swap.from.account;
+  const { account: fromAccount, currency: fromCurrency } = swapTransaction.swap.from;
   const fromAccountUnit = fromAccount && getAccountUnit(fromAccount);
   const estimatedFees = swapTransaction.status?.estimatedFees;
   const showSummaryValue = fromAccountUnit && estimatedFees && estimatedFees.gt(0);
+  const family = fromCurrency
```

Comment:
Since you already have access to the `parentAccount` and the `account`, you can just rely on the `mainAccount.currency.family` since it's for sure going to be a cryptocurrency without checking. Something like this:

```
const { account: fromAccount, parentAccount: fromParentAccount, currency: fromCurrency } = swapTransaction.swap.from;
const mainFromAccount = getMainAccount(fromAccount, fromParentAccount);
// family is mainFromAccount.currency.family
```